packages/gcloud-mcp/src/tools/research_gcloud_command.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js';
+import * as gcloud from '../gcloud.js';
+import { z } from 'zod';
+import { log } from '../utility/logger.js';
+
+export const createResearchGcloudCommand = () => ({
```

Comment:
When testing, there are still ANSI escape codes in the content that make the output pretty noisy.